convert2rhel/cert.py

```diff
@@ -75,4 +75,7 @@ def remove(self):
             os.remove(self._target_cert_path)
             loggerinst.info("Certificate %s removed" % self._target_cert_path)
         except OSError as err:
-            loggerinst.error("OSError({0}): {1}".format(err.errno, err.strerror))
+            if err.errno == 2:
```

Comment:
We could put a comment above this check to indicate that the `err.errno == 2` means that the file was not found

Reply:
yeah that sounds like a better way to handle this thank you!

Reply:
@Andrew-ang9 the conversation is resolved but it was never changed. Adding a comment stating the reason why it's skipped will be ideal, as currently if we encounter this further down the line we'd have no clue what is going on and would have to look up the return codes of certificate

And although we shouldn't log an error, adding a log at debug level is preferable rather than passing